GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestApp.xcodeproj/project.pbxproj

```diff
@@ -537,7 +537,9 @@
 			baseConfigurationReference = 7457A7438B37247584BF5723 /* Pods-GDTCCTWatchOSTestApp WatchKit Extension.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_COMPLICATION_NAME = Complication;
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = EQHXZ8M8AV;
```

Comment:
Should probably leave the singing info out. I normally check the automatically signing box and then uncheck it. Then xcode will clear your personal signing info.